### PR TITLE
Add Namada chain

### PR DESCRIPTION
**Pull request by ljoss17**
This PR adds the Namada chain

### CODE REVIEW

**Comment by JonathanLorimer** [comment]
CI is gonna be blocked on this pr https://github.com/informalsystems/cosmos.nix/pull/192 (I broke CI for PR's from forks, due to the way github's OIDC / token permissions work.

File: packages/namada.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  namada-src,
+}:
+pkgs.rustPlatform.buildRustPackage {
+  pname = "namada";
+  version = "v0.27.0";
+  src = namada-src;
+  nativeBuildInputs = with pkgs;
+    if stdenv.isLinux
+    then [pkg-config protobuf llvmPackages.libclang llvmPackages.libcxxClang clang]
+    else [darwin.apple_sdk.frameworks.Security];
```
Comment (JonathanLorimer) [inline packages/namada.nix:12]: I'm surprised linux is so different from mac, I guess the mac env has the correct llvm / c packages? Does it not need protobufs?

File: packages/namada.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "masp_note_encryption-1.0.0" = "sha256-NwiosHTdzzny+L5VtOBaIa7wia/yRlfiz/8f0pAHUZk=";
+      "sparse-merkle-tree-0.3.1-pre" = "sha256-B1ZEN4FZjV0x0Cqvx7AZjH9qhDMZYFPVJzg89dqWCv4=";
+      "tiny-bip39-0.8.2" = "sha256-TU+7Vug3+M6Zxhy6Wln54Pxc9ES4EdFq5TvMOcAG+qA=";
+      "tower-abci-0.11.1" = "sha256-KisZtsylvUymvV1TpDdGIiE7fSarcuD3I8oZ33BdKTU=";
+      "wasmer-2.3.0" = "sha256-Fd8ewAwslopjqUVoeHwSR/Zoh4Zm+Sdx8oksXmhLU20=";
+      "zcash_encoding-0.2.0" = "sha256-keuaoM/t1Q/+8JMemMMUuIo4g5I/EAoONFge+dyWGy0=";
+    };
+  };
+  doCheck = false;
+  cargoCheckCommand = "true";
```
Comment (JonathanLorimer) [inline packages/namada.nix:33]: I'm surprised you need both of these, I would assume `doCheck` would prevent the entire check phase.

File: packages/namada.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      "index-set-0.8.0" = "sha256-oxJfQdKnYiW5VbMPuukVyDY5n8mys31hYNrJF89nXhY=";
+      "ledger-namada-rs-0.0.1" = "sha256-qFL8LU7i5NAnMUhtrGykVfiYX1NodCNkZG07twyVrac=";
+      "masp_note_encryption-1.0.0" = "sha256-NwiosHTdzzny+L5VtOBaIa7wia/yRlfiz/8f0pAHUZk=";
+      "sparse-merkle-tree-0.3.1-pre" = "sha256-B1ZEN4FZjV0x0Cqvx7AZjH9qhDMZYFPVJzg89dqWCv4=";
+      "tiny-bip39-0.8.2" = "sha256-TU+7Vug3+M6Zxhy6Wln54Pxc9ES4EdFq5TvMOcAG+qA=";
+      "tower-abci-0.11.1" = "sha256-KisZtsylvUymvV1TpDdGIiE7fSarcuD3I8oZ33BdKTU=";
+      "wasmer-2.3.0" = "sha256-Fd8ewAwslopjqUVoeHwSR/Zoh4Zm+Sdx8oksXmhLU20=";
+      "zcash_encoding-0.2.0" = "sha256-keuaoM/t1Q/+8JMemMMUuIo4g5I/EAoONFge+dyWGy0=";
+    };
+  };
```
[the remark this answers is not in the document]
Comment (ljoss17) [reply packages/namada.nix:31]: Oh I will remove the `cargoSha256`, without `cargoLock` I had an error saying :

```
ERROR: The Cargo.lock contains git dependencies
       This is currently not supported in the fixed-output derivation fetcher.
       Use cargoLock.lockFile / importCargoLock instead.
```

[the remark this answers is not in the document]
Comment (JonathanLorimer) [reply packages/namada.nix:31]: Oh yeah, that makes sense (`cargoSha256` or any `*Sha256` you see in nix refers to fixed-output derivations, so hopefully that makes the error message make more sense.)

File: packages/namada.nix (hunk 1 above)
Comment (ljoss17) [reply packages/namada.nix:12]: Oh I still wasn't able to have it work on Linux so I still haven't tested on Mac. I suspect I might need to change the `protobuf` to be used for both.

File: packages/namada.nix (hunk 2 above)
Comment (ljoss17) [reply packages/namada.nix:33]: I see, should it also be removed here then, https://github.com/informalsystems/cosmos.nix/blob/af5652b30c3ecc130380934da2161fa9a8e6b3f5/packages/hermes.nix#L12?

Comment (JonathanLorimer) [reply packages/namada.nix:33]: Yup!

File: packages/namada.nix (hunk 1 above)
Comment (romac) [reply packages/namada.nix]: Maybe Nix uses the default LLVM toolchain present on Mac via Xcode/Command-Line Tools?

**Comment by romac** [comment]
Thank you so much for the support Jon 🙏

**Review by romac — APPROVED** [review]

**Pull request merged**